runtime/vm/jvminit.c

```diff
@@ -1823,7 +1823,8 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				IDATA argIndexIgnoreUnrecognizedOptionsEnable = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXIDLETUNINGIGNOREUNRECOGNIZEDOPTIONSENABLE, NULL);
 				IDATA argIndexIgnoreUnrecognizedOptionsDisable = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXIDLETUNINGIGNOREUNRECOGNIZEDOPTIONSDISABLE, NULL);
 
-				if (argIndexGcOnIdleEnable > argIndexGcOnIdleDisable) {
+				/* idle heap tuning is enabled only if -XX:+IdleTuningGcOnIdle or -Xtune:virtualized is set as VM options*/
```

Comment:
This should be limited to Java 9 only.  We don't want to change the default for this on Java 8.